core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/setoperators/UnionOperator.kt

```diff
@@ -0,0 +1,13 @@
+package ch.ergon.dope.resolvable.clause.model.setoperators
+
+import ch.ergon.dope.resolvable.clause.ISelectOffsetClause
+
+class UnionOperator(
+    leftSelect: ISelectOffsetClause,
+    rightSelect: ISelectOffsetClause,
+    duplicatesAllowed: Boolean = false,
+) : SetOperator("UNION", leftSelect, rightSelect, duplicatesAllowed)
+
+fun ISelectOffsetClause.union(select: ISelectOffsetClause) = UnionOperator(this, select)
+
+fun ISelectOffsetClause.unionAll(select: ISelectOffsetClause) = UnionOperator(this, select, true)
```

Comment:
please always add named parameters if you just pass a primitive value (true)

Reply:
done